functions.inc.php.php

```diff
@@ -11,7 +11,7 @@
 session_regenerate_id();
 if(!isset($_SESSION['csrf'])) csrf_gen();
 
-if(file_exists("db.inc") && file_exists("config.php")) unlink("config.php");
+if(file_exists("db.inc.php") && file_exists("config.php")) unlink("config.php");
 
 // Display basic website head section + body beginning
 function showHead($subname, $header = "Ambasadorzy Jawności") {
```

Comment:
@tradzik , to jest bot. Fakatycznie bywa omylny, ale podoba mi się, że czasem wzbudza refleksje. Nie musisz odnosić się do tych uwag.